ext/mysqlnd/config-win.h

```diff
@@ -15,32 +15,6 @@ This file is public domain and comes with NO WARRANTY of any kind */
 
 #include <stdint.h>
 
-#ifndef HAVE_INT8_T
-#define HAVE_INT8_T
-#endif
```

Comment:
Hmm, I doubt that these macros are defined on Windows platforms; I don't know whether this is relevant, though.

Reply:
Yes, they are defined - exactly here, and this PR removes those definitions, because the checks for those macros are also being removed. On Windows, there just was no configure-time check, and it was assumed the C99 integers are always available.

Reply:
Ah, right! Some external extensions may use these defines, but they probably can just be updated as well. Needs an entry in UPGRADING.INTERNALS, but otherwise removing this cruft looks good.

Reply:
Added some text to UPGRADING.INTERNALS.